cmd/yesiscan/main.go

```diff
@@ -59,16 +63,30 @@ var program string
 //go:embed .version
 var version string
 
+// autoConfigURI is set via -ldflags build time flags.
+var autoConfigURI string
+
+// autoConfigCookiePath is set via -ldflags build time flags.
+var autoConfigCookiePath string
+
 const (
 	// ConfigFileName is the name of the config file used to pull in all the
 	// various main settings that we want.
 	ConfigFileName = "config.json"
+
+	// MaxRedirects is the maximum number of redirects to allow for http
+	// download operations. The internal golang maximum of ten is too low
+	// for many situations. Firefox sets network.http.redirection-limit as
+	// 20.
+	MaxRedirects = 20 // do what firefox does
 )
 
 // CLI is the entry point for the CLI frontend.
 func CLI(program, version string, debug bool, logf func(format string, v ...interface{})) error {
 
 	flags := []cli.Flag{
+		&cli.StringFlag{Name: "auto-config-uri"},
+		&cli.StringFlag{Name: "auto-config-cookie-path"},
 		&cli.BoolFlag{Name: "quiet"},
 		&cli.StringFlag{Name: "regexp-path"},
 		&cli.StringFlag{Name: "config-path"},
```

Comment:
Is there a way for CLI flags to be self-documented? I didn't notice this before this review, but `--help` is not very helpful.

Reply:
I agree. This should be improved. Can you open an issue for it or send a patch please? Thanks!

Reply:
#47